packages/webpack-cli/lib/utils/prompt-installation.js

```diff
@@ -45,7 +50,7 @@ async function promptInstallation(packageName, preMessage) {
             process.exit(2);
         }
 
-        return utils.packageExists(packageName);
+        return packageName;
```

Comment:
I think we don't need check here, because `require` fails itself if we will have problem. 

Before we have `require.resolve(package)` and then `require()`, it doesn't make sense, because `require` call `require.resolve` internally

Reply:
Yes , we can skip it as we reach here after successful installation only.

P.S.
I guess this was added here to make sure `webpack` is avaliable before running CLI (talking about code at `bin/cli.js`) and as it is not being checked in `then` block.

Reply:
I see